web/src/cypress/support/form.ts

```diff
@@ -136,7 +136,18 @@ function fillFormField(
         .data('cyFallbackType')
 
       if (isSelect) {
-        if (value === '') return cy.get(selector).clear()
+        if (value === '') {
+          cy.get(selector).clear()
+
+          // clear chips on multi-select
+          el.parent()
+            .find('[data-cy="multi-value"] > svg')
+            .each(() => {
+              cy.get(selector).type(`{backspace}`)
+            })
+
+          return cy.get(selector).clear()
```

Comment:
I'm curious why `cy.get(selector).clear()` is being called both before and after clearing the chips– were you seeing test failures without?